client/src/components/TabbedContent.js

```diff
@@ -42,7 +44,12 @@ export class TabbedControls extends React.Component {
     );
   }
 }
-
+TabbedControls.defaultProps = {
+  disabled_message: {
+    en: "Disabled",
```

Comment:
I guess we could put this into `common_lang.yaml` but this text is never used other than here, so it really doesn't matter. Just thought I'd mention it

Reply:
noted;
I think this was a quick fix at the time, but unsure where else we'd put it